quantities.test/TemperatureTest.cs

```diff
@@ -8,6 +8,9 @@ namespace Quantities.Test;
 // For a lot of the values here, see: https://en.wikipedia.org/wiki/Conversion_of_units_of_temperature#Comparison_of_temperature_scales
 public sealed class TemperatureTest
 {
+    private static readonly Int32 horriblyLowPrecision = VeryLowPrecision - 1;
+    private static readonly Int32 abysmallyLowPrecision = VeryLowPrecision - 3;
```

Comment:
Note the loss of precision encountered due to the change.

Reply:
Could you change this to reference the `fullPrecision`? This would make it easier to trace back to the `Convenience` class.